### ISSUE TITLE
Combobox from `bits-ui`

### ISSUE THREAD

**Issue by IvanProg00** [huntabyte/shadcn-svelte#1579]
### Prerequisites

- [X] This feature already exists in [shadcn/ui](https://github.com/shadcn/ui) - if not, it won't be considered here so don't continue with your issue.

### Describe the feature

Currently, the [Combobox](https://next.shadcn-svelte.com/docs/components/combobox) component is implemented using [Popover](https://next.shadcn-svelte.com/docs/components/popover) and [Command](https://next.shadcn-svelte.com/docs/components/command) components. I understand that this approach was chosen because the [Combobox](https://www.bits-ui.com/docs/components/combobox) component was not available in `bits-ui`. However, now that it is available, it would be better to use it instead of two separate components.

Therefore, for the **Next Version**, it would be better to rely on the `bits-ui` component for the Combobox instead of relying on two separate components.

I think it would be a good idea to create a separate component for the combobox. Currently, to style the [Popover.Trigger](https://next.shadcn-svelte.com/docs/components/popover) component using Tailwind, I either have to create a separate class for it or assign different classes to each [Popover.Trigger](https://next.shadcn-svelte.com/docs/components/popover).

**Comment by amlcx** [comment]
You're right. Not to mention, at the moment, since the combobox in `shadcn-svelte` is based on two different components, it doesn't even have a link to the component's source code.
